### ISSUE TITLE
Dependencies for package check recursively bring in this package

### ISSUE THREAD

**Issue by hlapp** [ropensci/RNeXML#179]
For running `R CMD check` on Travis, the installation of dependencies includes those in the suggested  category, which includes `phylobase`. However, `phylobase` itself depends on `RNeXML`, and hence installing the `phylobase` package results in also installing RNeXML from whichever is the latest version on CRAN.

I don't know enough about the package loading intricacies in R whether that means phylobase functions will use the CRAN-installed version, or the current version of RNeXML being tested, or whether the latter will override the former by way of getting installed over it before anything is being run.

Either way, it's a rather confusing situation for judging what's happening, and generally speaking reciprocal dependency is probably best to be avoided unless it arguably can't be. So, do we really have to have code in this package that's dependent on phylobase, and which is that? I can't seem to find any, actually.

**Comment by cboettig** [comment]
Yeah, it's a confusing situation that should be avoided.  I think we can drop Suggest on phylobase -- I don't see it being used anywhere either.  (We had explored RNeXML->phylobase converters before but ended up punting on that issue;  I think that's handled on the phylobase side now via RNeXML).  Thanks for catching this.

**Comment by hlapp** [comment]
See #87. Indeed coercion to phylobase classes is handled there (fmichonneau/phylobase@5401a02ec14cb84919fc040de9f4cfb322a849f8). It seems to me that we can simply drop the dependency.
